lib/galaxy/model/migrate/versions/util.py

```diff
@@ -168,6 +181,7 @@ def drop_index(index, table, column_name=None, metadata=None):
             if not isinstance(table, Table):
                 assert metadata is not None
                 table = Table(table, metadata, autoload=True)
+            index = truncate_index_name(index, table.metadata.bind)
```

Comment:
```suggestion
            index_name = truncate_index_name(index, table.metadata.bind)
```

Reply:
(a) to keep it consistent with `add_index`; and
(b) to stay sane: one is Index, the other is its name

Reply:
Also, see line 188: do we want to log the original name or the truncated name? (i.e., `index` vs `index_name` - if we rename variable; otherwise we have no choice and original name is lost)

Reply:
original makes more sense.